curiouslearning/FeedTheMonsterJS
# fix: audio issue when no stone (FM-313)

# Changes
- Fix audio on drag issue when stone dont exists

# How to test
- Open the start screen of FTM.
- Tap or click any part of the start screen to proceed to the next screen
- Select and play any level
- While the stone is not in place or exists try to drag anywhere the screen
- Make sure theres no drag sound effect playing while stone is not loading

Ref: [FM-313](https://curiouslearning.atlassian.net/browse/FM-313)


[FM-313]: https://curiouslearning.atlassian.net/browse/FM-313?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
